pkg/customcost/pipelineservice.go

```diff
@@ -81,23 +82,18 @@ func getRegisteredPlugins(configDir string, execDir string) (map[string]*plugin.
 		configs[name] = &plugin.ClientConfig{
 			HandshakeConfig: handshakeConfig,
 			Plugins:         pluginMap,
-			Cmd:             exec.Command(execDir+"/"+name+".ocplugin."+version.Architecture, config),
+			Cmd:             exec.Command(execDir+"/"+name+".ocplugin."+runtime.GOOS+"."+version.Architecture, config),
```

Comment:
Could we pull out this duplicate string concatenation into a variable? Also, how is this string format determined?

Reply:
I refactored it to use a fmt string, that hopefully cleans it up. Since the plugins run as binaries, the executable name depends on the OS and processor. The idea is that the person would get the binary built for their machine from here: https://github.com/opencost/opencost-plugins/releases/tag/v0.0.3